pkg/users/account-create-dialog.js

```diff
@@ -284,20 +338,33 @@
 
         errs.user_name = validate_username(user_name, accounts);
 
+        const promises = [
+            password_quality(password, force_weak)
+                    .catch(ex => {
+                        errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
+                    })
+        ];
         if (!force_uid)
             errs.uid = validate_uid(uid, accounts, min_uid, max_uid, change);
 
-        return password_quality(password, force_weak)
-                .catch(ex => {
-                    errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
-                })
+        if (!force_home) {
+            promises.push(
+                validate_home_dir(state.home_dir)
+                        .catch(() => {
+                            errs.home_dir = cockpit.format(_("The home directory $0 already exists. Its ownership will be changed to the new user."), state.home_dir);
+                            state.home_dir_exists = true;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230324-000238-1c01f86d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)